internal/settings.go

```diff
@@ -165,31 +165,32 @@ func (ds *DialSettings) Validate() error {
 	return nil
 }
 
-// GetDefaultUniverseDomain returns the default service domain for a given Cloud
-// universe, as configured with internaloption.WithDefaultUniverseDomain.
-// The default value is "googleapis.com".
+// GetDefaultUniverseDomain returns the Google default universe domain
+// ("googleapis.com").
 func (ds *DialSettings) GetDefaultUniverseDomain() string {
-	if ds.DefaultUniverseDomain == "" {
-		return universeDomainDefault
-	}
-	return ds.DefaultUniverseDomain
+	return defaultUniverseDomain
```

Comment:
nit: I don't think we need this method, do we?

Reply:
It's an exported method, I didn't want to remove it.

Reply:
its in an internal package. If we need it fine, but we should also feel fine removing it and or making the const public instead